src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -262,27 +262,31 @@
    environment :- schema/Str
    jruby-service :- (schema/protocol jruby-protocol/JRubyPuppetService)
    request-tag :- (schema/maybe String)
-   last-updated :- (schema/maybe schema/Int)]
+   last-updated :- (schema/maybe schema/Int)
+   environment-class-cache-enabled :- schema/Bool]
```

Comment:
It might be good to update the docstring for this function to mention this setting and what it does to the response? (not blocker for merge)